brigade-worker/src/k8s.ts

```diff
@@ -566,14 +556,26 @@ function sidecarSpec(
     } as kubernetes.V1EnvVar);
   }
 
+  if (project.repo.token) {
+    spec.env.push({
+      name: "BRIGADE_REPO_AUTH_TOKEN",
+      valueFrom: {
+        secretKeyRef: {
+          key: "github.token",
+          name: secName
+        }
+      }
+    } as kubernetes.V1EnvVar);
+  }
+
   return spec;
 }
 
 function newRunnerPod(
   podname: string,
   brigadeImage: string,
   imageForcePull: boolean,
-  serviceAccount: string,
+  serviceAccount: string
```

Comment:
Very nit-picky but I prefer the trailing comma. So that we won't necessary introduce a bit noisy diff due to just adding a new parameter to this func.

```
function newRunnerPod(
    podname: string,
    brigadeImage: string,
    imageForcePull: boolean,
 -  serviceAccount: string
 +  serviceAccount: string,
 +  aBrandNewParam: string
```

Reply:
This is a result of running `yarn format`

Reply:
Got it. LGTM then. Thanks for the explanation 👍